### DISCUSSION TITLE
2021-08-04 Weekly Meeting 15 Agenda

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/omigroup#71] [Weekly Meetings]
Weekly Meeting 15 is happening on Wednesday 2021-08-04 from 9:00 AM PDT - 10:00 AM PDT

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR Discord](https://discord.gg/NJtT9grz5E).

## Agenda
- W3C Working Group Chair Election #69 
- OMI Weekly Meeting Discord Bot Update #63
- OMI Website Update #49
- OMI Virtual Space Update #44 
  - World Building / Exploration Meetup #70
- GLTF extensions sub-group update
- Demo Day
- Stellar Mesh #73

## Add Ons
*To add an agenda item please comment below*
- Let's have a conversation about our mission and vision! #72

**Comment by mrmetaverse** [comment]
Hi everyone! Me again :sweat_smile: 

Created #72 Draft Vision and Mission Statments for getting to the core of why we are here. 

Today an important question was asked during our weekly meeting. _What is our Vision?_

We often talk about it, but we could benefit from writing it down together. https://github.com/omigroup/OMI/issues/72

I propose that we take 10-15 min during our next meeting to look at these drafts, and have a discussion about these. I then propose we add our draft vision and mission to our readme, as I believe that will help instill an enhanced sense of pride and accountability for ourselves, clarity for the public, and foundational structure some members need to do their best work.

**Comment by mrmetaverse** [comment]
## :notebook: Notes for todays meeting are prepared :point_right: :  [here on hackmd](https://hackmd.io/@mrmetaverse/omi-080421)
